### ISSUE TITLE
Tekken 3 Missing TCB

### ISSUE THREAD

**Issue by wmvincent** [socram8888/tonyhax#9]
When trying to load Tekken 3, it shows a missing TCB error message.
Maybe set default values when they are missing in the SYSTEM.CNF file?

This post mentions the default values, not sure if it's accurate or not.
https://www.ngemu.com/threads/psx-boot-sequence.6773/

**Comment by socram8888** [comment]
Could you please share the contents of the SYSTEM.CNF file? Are they really missing or is the parser failing?

**Comment by hurda** [comment]
Tekken 3 (Europe) [SCES-01237]
```
BOOT = cdrom:\TEKKEN3\SCES_012.37;1
EVENT = 10
STACK = 801FFFF0
```

Tekken 3 (USA) [SLUS-00402]
```
BOOT = cdrom:\TEKKEN3\SLUS_004.02;1
EVENT = 10
STACK = 801FFFF0
```

**Comment by socram8888** [comment]
Thanks, I will modify it to use defaults if the parameters aren't specified.

**Comment by socram8888** [comment]
Fixed in e7a0f51a554587189b28785522b493d6d045909a

**Comment by wmvincent** [comment]
Wow, thanks hurda and socram8888!  That was fast.  Tried out the latest build and works great!
